src/Main.cpp

```diff
@@ -213,13 +282,17 @@ int main(int argc, char** argv) {
     static UartMessageSender s_uartMessageSender("uart_send", tskIDLE_PRIORITY + 1);
     static TCPMessageSender s_tcpMessageSender("uart_send", tskIDLE_PRIORITY + 1);
     static SPIMessageSender s_spiMessageSender("spi_send", tskIDLE_PRIORITY + 1);
+    static USBMessageSender s_usbMessageSender("usb_send", tskIDLE_PRIORITY + 1);
+    static USBMessageDispatcher s_usbMessageDispatcher("usb_dispatch", tskIDLE_PRIORITY + 1);
 
     s_bittybuzzTask.start();
-    s_uartDispatchTask.start();
+//    s_uartDispatchTask.start();
     s_tcpDispatchTask.start();
-    s_uartMessageSender.start();
+//    s_uartMessageSender.start();
     s_tcpMessageSender.start();
-    // s_spiMessageSender.start();
+//    s_spiMessageSender.start();
+    s_usbMessageSender.start();
+    s_usbMessageDispatcher.start();
```

Comment:
Will need to switch the ROS uart to the `IUSB` interface. Can be addressed in another PR, but let's just not forget it.

Reply:
USBMock has been added